libs/subliminal_patch/providers/whisperai.py

```diff
@@ -0,0 +1,83 @@
+from __future__ import absolute_import
+import logging
+
+from requests import Session
+
+from subliminal_patch.subtitle import Subtitle
+from subliminal_patch.providers import Provider
+from subliminal import __short_version__
+from subliminal.exceptions import ConfigurationError
+from subzero.language import Language
+from subliminal.video import Episode, Movie
+
+logger = logging.getLogger(__name__)
+
+
+class WhisperAISubtitle(Subtitle):
+    '''Whisper AI Subtitle.'''
+    provider_name = 'whisperai'
+    hash_verifiable = False
+
+    def __init__(self, language, video):
+        super(WhisperAISubtitle, self).__init__(language)
+
+        self.video = video
+
+    @property
+    def id(self):
+        return self.video.original_name
+
+    def get_matches(self, video):
+        matches = set()
+        matches.update(["title", "series", "season", "episode"])
```

Comment:
Make a conditional for video types (`Movie` and `Episode`). `["title", "series", "season", "episode"]` is exclusive of the `Episode` type.